src/Microsoft.SqlTools.Serialization/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.SqlTools.Credentials;
+using Microsoft.SqlTools.Credentials.Utility;
```

Comment:
why is serialization taking a dependency on Credentials assembly?  Serialization shouldn't use Credentials HostLoader unless we're planning to expose credentials APIs via serialization utility (I don't think we are).